code.js

```diff
@@ -5,3 +5,21 @@ function sum(a) {
     }
     return sum;
 }
+
+public class Main
+{
+    static int sum(int[] a) {
+	    int total = 0;
```

Comment:
The code still has errors! It seems like your code mixed with C programming language and Javascript!